src/Gemini.Modules.Inspector/Inspectors/NoUndoEditorBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Gemini.Modules.Inspector.Inspectors
+{
+    /// <summary>
+    /// This class is used as an editor base for values that do not support
+    /// undo / redo operations, for example write only variables.
```

Comment:
Yes, write only. It is basically producing an event with the value as the argument. It can not be read and does not make sense to undo. In any case, write only is just an example, I believe there are more cases where not having undo is necessary. Maybe we should just put that functionality into EditorBase ?